src/components/CardBox.js

```diff
@@ -8,18 +8,36 @@ const CardBox = ({
   showingSlideCardNum,
   slidingSize,
   cardInfos,
+  handleModal,
+  isModalVisible,
+  handleTransitionEnd,
 }) => {
   return (
     <StyledCardContainer
       hasBtn={hasButton}
       showingSlideCardNum={showingSlideCardNum}
       slidingSize={slidingSize}
+      onTransitionEnd={handleTransitionEnd}
     >
-      {cardInfos.map((cardInfo, idx) => (
-        <StyledCard key={idx}>
-          <Card cardInfo={cardInfo} showingSlideCardNum={showingSlideCardNum} />
-        </StyledCard>
-      ))}
+      {cardInfos.map((cardInfo, idx) =>
+        isModalVisible ? (
+          <StyledCard key={idx}>
+            <Card
+              cardInfo={cardInfo}
+              showingSlideCardNum={showingSlideCardNum}
+              isOpenModal={true}
```

Comment:
boolean값 true일때 줄여서 써주셔도 됩니다 👍 
```suggestion
              isOpenModal
```